servers/lib/src/config/config.service.ts

```diff
@@ -0,0 +1,50 @@
+import { readFileSync } from 'fs';
+import * as yaml from 'js-yaml';
+import { Injectable, Logger } from '@nestjs/common';
+import {ConfigValues, GitRepo } from './config.model.js';
+import resolveFile from './util.js';
+
+@Injectable()
+export default class Config {
+  private configValues: ConfigValues;
+  private logger: Logger;
+
+  constructor() {
+    this.logger = new Logger(Config.name);
```

Comment:
This Logger class has also been instantiated in `bootstrap.ts`. Is this the recommended nestjs way?

Reply:
Yes. The logger uses the Class name, so it is easier to see where the log comes from.